src/app/content/practiceQuestions/components/ShowPracticeQuestions.tsx

```diff
@@ -115,7 +118,8 @@ const ShowPracticeQuestions = () => {
             {
               selectedSectionHasPracticeQuestions && currentQuestionIndex === null
                 ? <IntroScreen />
-                : null
+                : typeof currentQuestionIndex === 'number' && section &&
```

Comment:
this whole conditional will need to be redone in the next card, but if this is what you need to get it working for now thats fine